### DISCUSSION TITLE
blog/stories-of-three-chinese-intellectuals

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#48] [Announcements]
# blog/stories-of-three-chinese-intellectuals

Two thousand seven hundred and fifty words

https://pathos.page/blog/stories-of-three-chinese-intellectuals

**Comment by savouer1** [comment]
对你们专业的事不熟，但以外行的视角看，我们的法律都是借鉴欧美的，基本上没有沿袭自己传统的。且目前法理学和法哲学的权威都是西方人，国内的法律专家都是啃人家书成神的，饮水思源都要祭下祖师爷。要谈神法和自然法，就离不开上帝和圣经。
西方的叙事框架已经深入人心，各领域都是如此。我们现在宣扬的社会主义或共产主义，也不是我们传统的叙事方式，且我们到目前为止自己都没有弄明白社会主义叙事该如何清晰的表达出来，这让别人怎么理解和接受。我感觉犯迷糊的不仅是专家们，底层人也很迷糊，只是不自知而已。简单说，要摒弃西方，得先把我们自己说明白；要是一直说不明白，那后来的学者还是会学西方那一套。

**Reply by Pathsis** [reply]
即使雷兄是外行，也可以看出我所要表达的意思，这些知识分子没能尽力穿透一时的现实迷雾，以更宽阔、更超越的视野观察和思考自己和这个世界，他们获得的是临时的，地域性的东西，但在他们离开这个世界时，却以为自己拿到了永恒的、普遍性的东西。这就是为什么我会对他们抱有一丝怜悯。

我们的实践虽然学习借鉴了西方的许多东西，但它不是对任何他人事业的继承。这完全是我们自己的事业，我们有自己的理论，自己的精神，自己的原则。像罗教授这种把亚伯拉罕看做他和我们的先知，看做我们法治事业的先知，我们必须说，我对学业不精但却以为很深沉的他感到可怜。

**Comment by calon** [comment]
为什么刚好以西方-非西方、欧美-中国的粒度来划分知识、精神、理论和原则呢？也可以是东亚的、东北的、珠三角区域的、大理的、东城区的、二马路的、道家的、王阳明的、木匠的、茶博士的…为什么要统一为中国的呢，明明其中还有许多不同之处。
在我看来，即使是继承自中国的传统，同样也是重重迷雾，且因为批判精神的缺失，更难以超越这片迷雾。不管是从哪方面继承，都需要从个人的角度批判、深思、践行然后再形成基础坚实的观念。
